examples/ds1307/ds1307.ino

```diff
@@ -16,12 +16,12 @@ void setup () {
 
   if (! rtc.isrunning()) {
     Serial.println("RTC is NOT running!");
+  }
     // following line sets the RTC to the date & time this sketch was compiled
```

Comment:
Then these comments should be indented one level less.

Reply:
Agreed